test_conformance/subgroups/subhelpers.h

```diff
@@ -354,25 +1233,55 @@ struct test
         clProgramWrapper program;
         clKernelWrapper kernel;
         cl_platform_id platform;
-        cl_int sgmap[2 * GSIZE];
+        cl_int sgmap[4 * GSIZE];
```

Comment:
This variable and the related parameters could (and probably should) be replaced with
```cpp
cl_int4 sgmap[GSIZE];
```
as this would make the code easier to reason about and correspond more explicitly to the `xy` kernel parameters.

This can be done later, too, if desired.